packages/shared/src/components/modals/award/GiveAwardModal.tsx

```diff
@@ -85,7 +99,7 @@ const IntroScreen = () => {
               bold
               color={TypographyColor.Primary}
             >
-              15 Awards given
+              {entity.numAwards || 0} Awards given
```

Comment:
Actually shouldn't render this section if 0/undefined :)

Reply:
7f1f0b5a6b392f87bea385956abb9686354227a0